### PR TITLE
feat(logs): add option to preserve `time` attribute

### PR DESCRIPTION
**Pull request by astencel-sumo**
Fixes https://github.com/SumoLogic/sumologic-kubernetes-collection/issues/3233

### CODE REVIEW

File: deploy/helm/sumologic/README.md [hunk 1]
```diff
@@ -55,6 +55,7 @@ The following table lists the configurable parameters of the Sumo Logic chart an
...
+| `sumologic.logs.container.keep_time_attribute`                                        | When set to `true`, preserves the `time` attribute, which is a string representation of the `timestamp` attribute.                                                                                                                                                                                                                                                   | `false`                                                                                                                                                                              |
```
Comment (dmolenda-sumo) [inline deploy/helm/sumologic/README.md:58]: I know this has been already messed up, but wouldn't we like to have this list sorted alphabetically?

File: deploy/helm/sumologic/conf/logs/collector/otelcol/config.yaml [hunk 2]
```diff
@@ -189,6 +189,13 @@ receivers:
         to: attributes["stream"]
         type: move
 
+{{ if .Values.sumologic.logs.container.keep_time_attribute }}
+      - id: move-time-attribute
+        from: body.time
+        to: attributes["time"]
+        type: move
+{{ end }}
+
```
Comment (aboguszewski-sumo) [inline deploy/helm/sumologic/conf/logs/collector/otelcol/config.yaml:198]: could you add a helm template test for that?

File: deploy/helm/sumologic/README.md (hunk 1 above)
Comment (astencel-sumo) [reply deploy/helm/sumologic/README.md:58]: Yes, ideally I'd expect to have this list sorted.

File: deploy/helm/sumologic/conf/logs/collector/otelcol/config.yaml (hunk 2 above)
Comment (astencel-sumo) [reply deploy/helm/sumologic/conf/logs/collector/otelcol/config.yaml:198]: :+1: done

**Pull request merged**